scripts/classes/JEquipment.gd

```diff
@@ -85,13 +86,70 @@ func get_item(item_uuid: String) -> JItem:
 	return null
 
 
+func get_output_data() -> Dictionary:
+	var output: Dictionary = {}
+
+	for slot in items:
+		if items[slot] != null:
+			var item: JItem = items[slot]
+			output[slot] = {"uuid": item.uuid, "item_class": item.item_class, "amount": item.amount}
```

Comment:
Possibly useful to add a `to_json` on `JItem`, not needed for now.

Reply:
You're right, it should have the to_json to match the flow

Reply:
fixed in:
https://github.com/jonathaneeckhout/jdungeon/pull/42